src/main/java/io/github/arrayv/sorts/distribute/TimeSort.java

```diff
@@ -35,25 +36,20 @@ of this software and associated documentation files (the "Software"), to deal
  *
  */
 
+@SortMeta(
+    name = "Time",
+    category = "Distribution Sorts",
+    showcaseName = "Time Sort, Mul 10",
```

Comment:
default time is specified here, but default base isn't specified in Base-N Max Heap

Reply:
I'm planning to change the base in `showcaseName` to be dynamic at runtime later (which will resolve this).

Reply:
but for now?

Reply:
It's specified everywhere except Base-N Max Heap. Base-N Max Heap is the exception, not Time.

Reply:
well then change that I suppose